Parser/StarmonDSTParse.m

```diff
@@ -35,6 +35,14 @@
 % along with this program.
 % If not, see <https://www.gnu.org/licenses/gpl-3.0.en.html>.
 %
+try
+    sample_data = StaroddiParser(filename,mode);
+    return
+catch
+
+end
```

Comment:
is this really the best way to do this? bypassing the entire code? couldn't this be done in the function calling this parser?
otherwise, it would be great to add some comment above the try catch about the reasons why all the rest of the code is bypassed. And maybe a printf or something in after the catch to mention the default parser is used?

Reply:
> Is this really the best way to do this? bypassing the entire code? couldn't this be done in the function calling this parser?

It's the most transparent way for the end-users. Users and their configurations (toolbox and db's) are sometimes hardcoded for the older parser names. Moving to a different name would not allow backward compatibility - I discussed yesterday with gg.

>t would be great to add some comment above the try catch about the reasons why all the rest of the code is bypassed. And maybe a printf or something in after the catch to mention the default parser is used?

Yes, I'll put a msg on the exception. I was thinking that the user should not be aware of the fail but if we don't print anything we will never know if someone had a problem with the new parser.

The right way is to completely remove the entire code outside the try and just fail miserably.  Although we got a good test coverage for this parser and the old parser is pretty much deprecated, I'm just fencing a bit high here until q3 release.